src/carousel.css.ts

```diff
@@ -137,8 +137,10 @@ export const carousel = style([
     },
 ]);
 
+const responsiveLayoutSideMargin = fallbackVar(responsiveLayoutVars.sideMargin, '0px');
```

Comment:
this is needed because there is a story in webapp that loads a carousel outside a ResponsiveLayout, and in that case the sideMargin css var is not defined. This fallbacks to 0px when not defined and makes the carousel component more resilient